css/style.css

```diff
@@ -0,0 +1,81 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+  font-size: 16px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+#title{
+  margin:2rem 0 2rem 0;
+  font-weight: bolder;
+  font-size: 2rem;
+}
+#main{
+  background-color: #d3d3d3;
+  text-align: center;
+  padding: 1rem;
+  margin: 20px 8px;
+  border-radius: 5px;
+}
```

Comment:
- [ ] I think here also you didn't mention the reason it would be great if you mention the reason here why you are asking to add a new space? as I said above maybe it is obvious to you but not to others.💪🏻